src/plugins/displayLayout/components/DisplayLayout.vue

```diff
@@ -166,7 +166,7 @@ export default {
     },
     computed: {
         gridSize() {
-            return this.domainObject.configuration.layoutGrid;
+            return this.domainObject.configuration.layoutGrid.map(item => Number(item));
```

Comment:
Could just pass the Number function to map and eliminate the arrow function. `layoutGrid.map(Number)`.

Reply:
True! DOH